[docs] Quickstart for ExpressionEngine (#4777)
ddev/ddev: docs/content/users/quickstart.md

        ddev launch
        ```

=== "ExpressionEngine"

    ## ExpressionEngine

>     === "ExpressionEngine Zipfile Download"

Reviewer: Every dictionary I found agrees it’s two words, and it wouldn’t hurt to match ExpressionEngine’s own docs here:

```suggestion
    === "ExpressionEngine ZIP File Download"
```

From [this page](https://docs.expressionengine.com/latest/installation/installation.html):

![Screen Shot 2023-03-21 at 06 04 36 PM@2x](https://user-images.githubusercontent.com/2488775/226775600-7e5d0bae-1edf-491d-a3ab-6a31c435e1b0.png)

>         Download the ExpressionEngine code from [ExpressionEngine.com](https://expressionengine.com/), then:

Reviewer: Clarify we’re running things and link back to download+install instructions since they live in a different place for the Git install instructions:

```suggestion
        Download the ExpressionEngine code from [expressionengine.com](https://expressionengine.com/), then follow the [install instructions](https://docs.expressionengine.com/latest/installation/installation.html) by running the following:
```

Author: The odd thing here is that the DDEV instructions are far simpler and easier than the expressionengine.com instructions. You don't have to do most of the things when you use DDEV.

        ```bash
        mkdir my-ee && cd my-ee
        unzip /path/to/ee-zipfile.zip
        ddev config # Accept the defaults
        ddev start

>         ddev launch /admin.php

Reviewer: Explain what this step does so we can refer to it below:

```suggestion
        ddev launch /admin.php # Open installation wizard in browser
```

        ```

>         In the `admin.php` setup, use "db" for the "DB Server Address", "DB Name", "DB Username", and "DB Password"

Reviewer: Continue with the “installation wizard” language from above for context, use backticks to signify a literal string and (hopefully) avoid any confusion with quotes, and italicize UI strings for consistency with treatment elsewhere in the docs:

```suggestion
        When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
```

        Visit your site.

    === "ExpressionEngine Git Checkout"

>         ```bash

Reviewer: Provide some context:

```suggestion
Follow the instructions in ExpressionEngine’s Git repository [README.md](https://github.com/ExpressionEngine/ExpressionEngine#how-to-install):

        ```bash
```

Author: I like the context. Mostly you can't just follow the instructions.

Reviewer: I agree that the instructions here are much more straightforward, and it seems to me like they’re following the general steps anyway—the links out aren’t hard requirements (since the steps are immediately detailed below) but only providing context.

Author: I added the links, just with "based on"

Reviewer: That reads better than my suggestion, thank you!

        git clone https://github.com/ExpressionEngine/ExpressionEngine # for example
        cd ExpressionEngine
        ddev config # Accept the defaults
        ddev start
        ddev composer install
        touch system/user/config/config.php
        echo "EE_INSTALL_MODE=TRUE" >.env.php
        ddev start

>         ddev launch /admin.php  # Use DB Server = "db", everything else "db"

Reviewer: Stay consistent with previous method:

```suggestion
        ddev launch /admin.php  # Open installation wizard in browser
```

        ```

> 

Reviewer: Again be consistent with the first install method since the ending steps are identical:

```suggestion

When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
```
